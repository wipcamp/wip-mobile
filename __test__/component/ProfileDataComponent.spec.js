/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import ProfileData from '../../components/ProfileDataComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <ProfileData
                title="Test"
                detail="Test"
            />
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})