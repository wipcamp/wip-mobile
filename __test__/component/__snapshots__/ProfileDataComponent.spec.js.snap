// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
      },
      null,
      Object {
        "backgroundColor": "#FFF",
      },
    ]
  }
>
  <View
    style={
      Object {
        "flex": 0.2,
      }
    }
  />
  <View
    style={
      Object {
        "flex": 0.3,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontFamily": "Kanit",
          },
          Object {
            "fontSize": 16,
          },
          Object {
            "textAlign": "left",
          },
        ]
      }
    >
      Test
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 0.5,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontFamily": "Kanit",
          },
          Object {
            "fontSize": 16,
          },
        ]
      }
    >
      : 
      Test
    </Text>
  </View>
</View>
`;
